source/install-guide/configuring-database.rst

```diff
@@ -4,32 +4,105 @@
 Configuring the database
 ########################
 
-This section describes how to configure the database connection, if required.
+Introduction
+------------
+
+GeoNetwork uses a database to persist aspects such as metadata records, privileges and configurations.
+The database default structure is created by the application on initial startup. Subsequent releases of GeoNetwork 
+will update the database structure automatically. For this reason the database user initially needs create privileges on the database. 
+A number of database dialects are supported; :command:`PostGreSQL`, :command:`Oracle`, :command:`SQL Server`, :command:`H2`.
+This section describes various options to configure the database connection.
+
+H2 database
+-----------
 
 By default, an `H2 <http://www.h2database.com/html/main.html>`_ database is configured
 and created when the application first starts. The H2 database named ``gn.h2.db``
 is created:
 
-* In the ``jetty`` folder of the GeoNetwork application folder when using the :ref:`ZIP distribution <installing-from-zip>`;
+* In the ``jetty`` folder of the GeoNetwork application folder when using the :ref:`ZIP distribution <installing-from-zip>`.
 
 * In the ``bin`` folder of Tomcat when deploying the :ref:`WAR <installing-from-war-file>` on Tomcat (started using ``startup.sh``).
 
 .. note::
     You don't **need** to configure the database if you are happy with the local H2 database.
-    Only change the configuration if you wish to store the data on a remote database.
+    Only change the configuration if you wish to store the data on a remote database. 
+
+Configuring a database via config files
+---------------------------------------
+
+The database dialect is configured in :file:`/WEB-INF/config-node/srv.xml`. Uncomment the dialect to use. 
+
+A jdbc driver is included for PostGreSQL and H2. Other dialects require a jdbc driver to be installed.
+Download the jdbc library for the dialect and place it in ``/WEB-INF/lib`` or in the tomcat lib folder.
```

Comment:
If you update to the latest geotools 24.x (see https://github.com/geonetwork/core-geonetwork/pull/5007) the oracle jdbc driver is now included (since it is finally free to distribute and available in a maven repository).

There is no reason to ask folks to download a jdbc driver anymore....

Reply:
thanx, for sql server you still need to download a driver though